projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-settings-page.php

```diff
@@ -42,25 +61,25 @@ function page_render() {
 		);
 		$noscript_notice = str_replace(
 			'#TEXT#',
-			esc_html__( "Turn on JavaScript to unlock Jetpack's full potential!", 'jetpack' ),
+			esc_html( __( "Turn on JavaScript to unlock Jetpack's full potential!", 'jetpack' ) ),
```

Comment:
Why did you change this one from `esc_html__()` to `esc_html( __() )`, and the two below in the opposite direction?

Reply:
Uh, that was a mistake that happened while I was experimenting with some alternative ways of escaping this. Thanks for catching this.

Reply:
Fixed this in fbe1c74fe3df5767e14c464f13769a53efac862b